src/Components/PresentationalComponents/CVEPageDetailsDescription/CVEPageDetailsExpandable.js

```diff
@@ -0,0 +1,53 @@
+import { Text, TextContent } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+
+const CVEPageDetailsExpandable = props => {
+    const [isDescriptionTruncated, toggleDescriptionTruncated] = useState(true);
+    const [isDescriptionTooLong, toggleDesciptionTooLong] = useState(false);
+    const [truncatedDescription, setTruncatedDescription] = useState('');
+
+    const { characterLimit, showMoreLabel, showLessLabel, content } = props;
+
+    useEffect(() => {
+        if (String(content).length > characterLimit) {
+            toggleDesciptionTooLong(true);
+            setTruncatedDescription(props.content.slice(0, characterLimit - 1) + '…');
```

Comment:
since you destructured the variable `content ` you can use :) 
```suggestion
            setTruncatedDescription(content.slice(0, characterLimit - 1) + '…');
```